### DISCUSSION TITLE
Connection is attempted on SSL port with http and failing.

### DISCUSSION THREAD

**Discussion by GoodMew** [nextcloud/all-in-one#2951] [Questions] [answered]
### Steps to reproduce
1. Start up Nextcloud AIO in Docker
2. Navigate to https://localhost:8080/containers
3. Submit my.domain for New AIO Instance
4. Receive error "Domain does not point to this server or the reverse proxy is not configured correctly. See the mastercontainer logs for more details. ('sudo docker logs -f nextcloud-aio-mastercontainer')" however DNS is configured correctly and ports are open.
5. Checked the logs.

### Expected behavior 
Connection is attempted to either https://my.domain:443 or http://my.domain:80

### Actual behavior 
Connection is attempted to http://my.domain:443 which is an SSL port on a non SSL domain, which I assume is why it fails with the following error:
2023-07-12 12:53:57 NOTICE: PHP message: The response of the connection attempt to "http://my.domain:443" was: 
2023-07-12 12:53:57 NOTICE: PHP message: Expected was: d6bf2ca5add10ab11efd599a7702d2b9950956e540879048
2023-07-12 12:53:57 NOTICE: PHP message: The error message was: Empty reply from server

### Host OS 
Windows 11

#### Nextcloud AIO version 
6.2.1

**Comment by szaimen** [comment, marked as answer]
Hi, do you use cloudflare or a firewall that possibly blocks http access on port 443? Because the default domain check indeed uses http on port 443 but that should usually work if there is no external service that blocks this.

**Reply by GoodMew** [reply]
I do use cloudflare, I have their proxy disabled, so it's set to "only DNS"
Thanks for pointing me in the right direction! <3

**Reply by GoodMew** [reply]
So I've been googling for 30 minutes but am just getting more confused… I'm not trying to do anything fancy or weird. I own a domain I bought through Hostinger, they told me to use Cloudflare to connect the domain to my website. I followed the directions in NextCloud and disabled Cloudflare's proxy.
What do I need to do here? I only want NextCloud on my server, do I need to not use Cloudflare? What should I use?
I am just looking for the simplest and most straightforward setup...

**Reply by szaimen** [reply]
You could try to use a different domain provider

**Reply by purejosh** [reply]
Your suggestion is to change domain providers? That's not a simple setting you can just change.
